### ISSUE TITLE
Append doesn't seem to work

### ISSUE THREAD

**Issue by avinashsonee** [d4software/QueryTree#89]
Hi,

I was trying out QueryTree Advanced Query Builder for a PostgreSQL database. I have added 2 filter conditions to a data table and both the filters seems to work fine. And then I was trying to append these 2 filter nodes using an Append node but I see an error on the Append node.

Append seems to work fine when only either of the Filter nodes are connected to it but fails as soon as both the Filter nodes are connected.

filter2 > select2 
![image](https://user-images.githubusercontent.com/341604/68057131-087f1000-fd1b-11e9-933e-887c2878401f.png)

filter3 > select3
![image](https://user-images.githubusercontent.com/341604/68057160-17fe5900-fd1b-11e9-85b2-8e2d0c5d867a.png)

filter2 > select2 > append2
![image](https://user-images.githubusercontent.com/341604/68057264-64e22f80-fd1b-11e9-8d09-f4ee4f8df2a3.png)

ERROR!
filter2 > select2 > (append2) < select3 < filter3
![image](https://user-images.githubusercontent.com/341604/68057291-775c6900-fd1b-11e9-914d-e58ab8baaa31.png)

The columns are same in both the filters. There is data in each and every cell (row x column) of both the filters.

I tried with enabling and disabling the "Include Non-matched Columns" option of the Append node.

Let me know if you need any more details.

Thanks for building this awesome tool

**Comment by d4nt** [comment]
@avinashsonee thanks for raising this, and the detailed explanation. I was able to reproduce it and have committed a fix to the develop branch (c9a78c9). Are you able to test it from that branch before I merge it into master?

**Comment by avinashsonee** [comment]
@d4nt this works. thanks for the quick fix. This can be merged to master although I haven't tested it extensively.

**Comment by avinashsonee** [comment]
@d4nt I also don't see the scheduled reports to be actually working.
I've configured the settings as per https://github.com/d4software/QueryTree/blob/develop/docs/mail.md
I've tried scheduling a simple report at multiple different times but nothing worked. I didn't receive a mail.
Also the server was always running while checking this. In the server logs too, I didn't see any info about scheduled emails being sent or any kind of error related to that.
Should I raise a separate issue for this?
